ngraph/frontend/paddlepaddle/src/op/flatten_contiguous_range.cpp

```diff
@@ -0,0 +1,59 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <ngraph/opsets/opset6.hpp>
+#include "flatten_contiguous_range.hpp"
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph {
+namespace frontend {
+namespace pdpd {
+namespace op {
+
+NamedOutputs flatten_contiguous_range (const NodeContext& node) {
+    auto data = node.get_ng_input("X");
+
+    PartialShape input_shape = data.get_partial_shape();
```

Comment:
Please express in the graph form instead of relying on the known shape and rank. You can see how onnx importer did it: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/frontend/onnx_import/src/op/flatten.cpp

Reply:
Thanks for advise, will fix it